terraform/modules/quorum-cluster-region/quorum-cluster-region-backup-event.tf

```diff
@@ -0,0 +1,399 @@
+
+# This is to be placed in quorum-cluster-region
+
+provider "archive" {}
+provider "random" {}
+provider "null" {}
+
+data "local_file" "private_key" {
+  filename = "${var.private_key_path}"
+}
+
+resource "aws_s3_bucket_object" "object" {
+  count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+  depends_on = ["data.aws_kms_ciphertext.EncryptSSHKey", "local_file.EncryptedSSHKey"]
+  bucket = "${aws_s3_bucket.quorum_backup.id}"
+  key    = "${var.enc_ssh_key}"
+  source = "${var.enc_ssh_path}-${var.aws_region}"
+}
+
+resource "aws_sns_topic" "BackupEvent" {
+  count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+  name_prefix = "BackupLambda-${var.network_id}-${var.aws_region}-"
+}
+
+resource "aws_cloudwatch_event_rule" "BackupTimer" {
+  count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+  name_prefix = "BackupLambda-${var.network_id}-${var.aws_region}-" 
+  schedule_expression = "${var.backup_interval}"
+}
+
+resource "aws_cloudwatch_event_target" "sns" {
+  count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"  
+  rule      = "${aws_cloudwatch_event_rule.BackupTimer.name}"
+  target_id = "SendToSNS"
+  arn       = "${aws_sns_topic.BackupEvent.arn}"
+}
+
+resource "aws_sns_topic_subscription" "BackupLambda" {
+  count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+  topic_arn = "${aws_sns_topic.BackupEvent.arn}"
+  protocol  = "lambda"
+  endpoint  = "${aws_lambda_function.BackupLambda.arn}"
+}
+
+# Allow the SNS to trigger the backup lambda
+resource "aws_lambda_permission" "BackupLambda" {
+  count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+  statement_id = "AllowExecutionFromSNS-BackupLambda-${var.network_id}-${var.aws_region}"
```

Comment:
nit: this applies to the whole page at least. Align your `=` horizontally among variable declarations that are on adjacent lines.